### PR TITLE
Incorporate Filesystems and update diagrams

### PR DESCRIPTION
**Pull request by paulo-valim**
Screenshot 2023-08-10 at 07 38 53

### CODE REVIEW

File: src/pages/teams.astro [hunk 1]
```diff
@@ -133,6 +134,30 @@ const future = false;
...
+          <span class="text-gray-200 mt-6 sm:mt-8"
+            >Control access to shared secrets on a per-notebook basis, creating
+            a secure working environment. Livebook Teams automatically
+            replicates these settings across your team, using your shared team
+            encryption key to determine when a notebook is allowed to access
+            specific shared secrets. Empower your team to protect sensitive
+            information without the need to constantly manage permissions
+            manually.</span
+          >
+        </div>
```
Comment (josevalim) [inline src/pages/teams.astro:160]:
```suggestion
        <div class="m-auto max-w-2xl flex flex-col items-center">
          <img
            class="sm:max-w-sm"
            src={featureFilesystem}
            alt="Feature: Synchronized file storage access"
            width="571"
            height="247"
          />
          <h2
            class="text-gray-50 text-2xl md:text-3xl leading-tight mt-8 sm:mt-10"
          >
            Synchronized file storage access
          </h2>
          <span class="text-gray-200 mt-6 sm:mt-8"
            >Share meaningful data and resources across all team members.
            Livebook Teams automatically synchronizes your cloud storage
            credentials (such as Amazon S3, Google Cloud, etc) across team members,
            making it easier than ever to write and share notebooks that automate
            and build insights on data.</span
          >
        </div>
```

File: src/pages/teams.astro [hunk 2]
```diff
@@ -133,6 +134,27 @@ const future = false;
...
+            class="text-gray-50 text-2xl md:text-3xl leading-tight mt-8 sm:mt-10"
+          >
+            Synchronized file storage access
+          </h2>
+          <span class="text-gray-200 mt-6 sm:mt-8"
+            >Share meaningful data and resources across all team members.
+            Livebook Teams automatically synchronizes your cloud storage
+            credentials (such as Amazon S3, Google Cloud, etc) across team members,
+            making it easier than ever to write and share notebooks that automate
+            and build insights on data.</span
```
Comment (josevalim) [inline src/pages/teams.astro:155]: @hugobarauna , I gave a first pass. Feel free to do your own and merge when ready. :)

Comment (hugobarauna) [reply src/pages/teams.astro:155]: @josevalim Will do it now.

**Comment by hugobarauna** [comment]
@josevalim 

In that image:

image

What do you think of changing the wording from "filesystem" to "file storage"?

The problem is that when I think of a filesystem, I think of a regular local filesystem from an operating system. So I was thinking about changing that name to one that can encompass both S3 and a regular local filesystem.

**Comment by hugobarauna** [comment]
@josevalim 

> We call them filesystems in Livebook. We would have to change it there but I am not sure if it is worth it. 

I see, ok. I prefer file storage, but let's keep it that way to avoid the cost of changing it now.

> For example, we may support GitHub repos in the future, what term would best describe it?

Not sure, it depends on how we'd going to use that.

If it's a place where one would save their notebook source files, maybe notebook storage or file storage (more generic).

But if it's a place to save files used by notebooks, I'd say file storage.

**Comment by josevalim** [comment]
@hugobarauna perhaps the source of confusion is that you can navigate them using the filesystem component, as if it was a file system.

**Comment by hugobarauna** [comment]
@josevalim 

Given our call today and that some nomenclature around the UX/UI of Livebook Files may change, I'll wait a bit more before continuing to work on that copy.

For example, if we change the wording from "File systems" to "File storages," as we said, the image here may need to change as well, and maybe the copy.

Also, I realized I prefer to use the [feature the Ale is implementing](https://github.com/orgs/livebook-teams/projects/4?pane=issue&itemId=34796878) before writing this copy. That said, if we need to merge that branch before this, I'm also ok with that.

**Comment by jonatanklosko** [comment]
If we say that Livebook Teams allows for sharing file storages, it may be easier to confuse it with us actually providing the storage. It is certainly subjective, but in my mind "File system" doesn't scream "local", especially in the context of sharing. Using the word storage in explanations sounds good, I'm just not sure about rebranding filesystems as storage.

**Comment by josevalim** [comment]
To summarize:

1. File storage is the simpler term
2. However, adding "File storage" can be read as confusing
3. Therefore, we should change both the name and the verb, e.g. "Connect new file storage"

**Comment by josevalim** [comment]
We must say in the description we don't get access to your data because all credentials are encrypted. Perhaps that's the most important aspect of this item.

**Comment by hugobarauna** [comment]
> We must say in the description we don't get access to your data because all credentials are encrypted. Perhaps that's the most important aspect of this item.

just made a new version of that copy considering that idea and also adding a few more improvements

CleanShot 2023-09-29 at 17 33 23

@josevalim 

if this is good enough for you, I think we can merge and deploy this PR.

File: src/pages/teams.astro [hunk 3]
```diff
@@ -133,6 +134,29 @@ const future = false;
...
+            Synchronized file storage access
+          </h2>
+          <span class="text-gray-200 mt-6 sm:mt-8"
+            >Effortlessly share and access files stored in cloud services like
+            Amazon S3 across your team, eliminating the need for manual
+            credential sharing. Livebook Teams ensures team members can run
+            notebooks relying on shared data without sacrificing security or
+            convenience. Moreover, Livebook Teams never gets access to your
+            storage credentials; they're encrypted to ensure that only your team
+            can use them.
```
Comment (josevalim) [inline src/pages/teams.astro:157]:
```suggestion
            >Effortlessly share notebooks and access data stored in cloud services like
            Amazon S3 across your team, eliminating the need for manual
            credential sharing. Moreover, Livebook Teams never gets access to your
            storage credentials; they're encrypted to ensure that only your team
            can use them.
```

I removed the sentence in the middle, as it was pretty much repeating the other ones (the first sentence is about convenience, the last one about security).

**Review by josevalim — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**